apps/myaccount/src/components/profile/profile.tsx

```diff
@@ -909,6 +909,7 @@ export const Profile: FunctionComponent<ProfileProps> = (props: ProfileProps): R
                             <List.Content floated="right">
                                 { !CommonUtils.isProfileReadOnly(isReadOnlyUser)
                                 && schema.mutability !== ProfileConstants.READONLY_SCHEMA
+                                && schema.name !== "userName"
```

Comment:
shall we define and use constant here?

Reply:
Addressed with 805a8c2a797d6cd73e2d323e509cc18c0cee8f80 in https://github.com/wso2/identity-apps/pull/3041